src/o365/yammer/commands/report/report-groupactivityfilecounts.ts

```diff
@@ -0,0 +1,38 @@
+import commands from '../../commands';
+import PeriodBasedReport from '../../../base/PeriodBasedReport';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+class YammerReportGroupActivityCountsCommand extends PeriodBasedReport {
```

Comment:
Renamed from `YammerReportGroupActivityCountsCommand ` to `YammerReportGroupsActivityCountsCommand ` to match the issue description.